.github/workflows/ci-containers.yaml

```diff
@@ -176,6 +177,7 @@ jobs:
         id: docker_build
         uses: docker/build-push-action@v4
         with:
+          build-args: "vivo_base_path=/vivo"
```

Comment:
It might be better just to make this the default in the Dockerfile - we can still make it explicit here but then if people look just at the Dockerfile it is obvious.